demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -255,19 +268,11 @@ def ensure_unique_pack_dir_name(self, pack_dir: str) -> str:
 
     def unpack_contribution_to_dst_pack_directory(self) -> None:
         """Unpacks the contribution zip's contents to the destination pack directory and performs some cleanup"""
-        if self.contribution:
-            shutil.unpack_archive(
-                filename=self.contribution, extract_dir=str(self.working_dir_path)
-            )
-            # remove metadata.json file
-            Path(self.working_dir_path, "metadata.json").unlink()
-        else:
-            err_msg = (
-                "Tried unpacking contribution to destination directory but the instance variable"
-                ' "contribution" is "None" - Make sure "contribution" is set before trying to unpack'
-                " the contribution."
-            )
-            raise TypeError(err_msg)
```

Comment:
We made `self.contribution` a mandatory attribute to the `ContributionConverter` constructor, see https://github.com/demisto/demisto-sdk/blob/ee9ec03d5589d3ed94df37618b4066b0ae3683ac/demisto_sdk/commands/init/contribution_converter.py#L160. I changed `self.contribution` to be mandatory because it holds the path to the contribution zip generated by the server. It is always defined in the contribution management flow. 
As a result of making `self.contribution` mandatory, we can remove this check and `if self.contribution` from all places within the class making it a bit cleaner.